csv2wiki/__init__.py

```diff
@@ -751,7 +752,7 @@ def _save_page(self, page_title, text):
             # Again, klugey, but we want easy page-boundary visibility.
             self._dry_run_out.write("\n" + "#" * 78 + "\n\n")
 
-    def _do_skel(self, skel, page_title, row, row_num):
+    def _do_skel(self, skel, row):
```

Comment:
Thanks for simplifying this.  I see why `page_title` went away; separately, I see that `row_num` went away because it wasn't being used at all in the first place!  I must have left it there from something else.  Glad you cleaned this up.